pkg/satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -53,4 +53,30 @@ read one (
 )
 create company ( )
 update company ( where company.id = ? )
-delete company ( where company.id = ? )
\ No newline at end of file
+delete company ( where company.id = ? )
+
+model project (
+    key id
+
+    field id                   blob
+    field user_id              user.id   cascade
+
+    field name                 text ( updatable )
+    field description          text ( updatable )
+    field is_agreed_with_terms bool ( updatable )
```

Comment:
`terms_accepted`, maybe we want to make it as a version `int`, because we might need to update the terms. @jtolds?

Reply:
yeah that's a good idea @crawter

Reply:
ok, in that case i'll make some updates here @jtolds , @egonelbre